### ISSUE TITLE
new function check devices config did not identical

### ISSUE THREAD

**Issue by cynthiabour** [automatedchemistry/flowchem#130]
The current config file will still work if same address of two Elite11 was provide. It will be good to check config file before run

**Comment by JB-Wolf** [comment]
If I remembe correctly, this should fail from the device side. In my opinion, this is also sufficient/fine. If the thing fails because of a user error, the thing fails. It is impossible to prevent all user errors. Eg the pump with address 0 needs to be connected via usb physically. This can not be checked via software, just by initializing. As long as an error is raised, IMHO this behaviour is fine

**Comment by cynthiabour** [comment]
no. with the current method we have. it did not shows error. It might need more test.

**Comment by JB-Wolf** [comment]
So it normally initialises? everything and leaves you quietly in the end without anything?
Then I agree, we need a check
Our PumpIO object could hold a daisy_chain object with pump_numbers already connected. This could be checked against when creating a certain pump. With this, later additions and even deletions on initialised code are possible, whereas checking the config file just prevents intial problems

**Comment by cynthiabour** [comment]
yes and shows all devices connected. I only observed it once, so it need more investigation

**Comment by cynthiabour** [comment]
Please also check with ml600 @JB-Wolf
